src/common/const.ts

```diff
@@ -140,11 +142,13 @@ export const FIXED_DEVICE_CLASS_ICONS = {
   pressure: mdiGauge,
   reactive_power: mdiFlash,
   signal_strength: mdiWifi,
+  speed: mdiSpeedometer,
   sulphur_dioxide: mdiMolecule,
   temperature: mdiThermometer,
   timestamp: mdiClock,
   volatile_organic_compounds: mdiMolecule,
   voltage: mdiSineWave,
+  // volume: TBD, => no well matching icon found
```

Comment:
Not ideal, but maybe `mdiCupWater` as default?

Reply:
A very quick browse of the use `VOLUME_*` constants shows that, where the context is clear, it's mostly either of:
- gas
- water
- vehicle fuel
For gas, there's already a gas device class of course, so it's pretty much either an amount of water or vehicle fuel

The cup is not a great fit for vehicle fuel.

How about just doing this for the default:
![image](https://user-images.githubusercontent.com/14281572/192721831-ba612ecf-17cf-48af-8aaa-5b5fb35d49e7.png)

Reply:
There's also:
![image](https://user-images.githubusercontent.com/14281572/192722527-f2688ceb-59fb-4537-aa13-7ed10ca33ca3.png)

Reply:
I like that `mdiCarCoolantLevel` shows inside a container.
![image](https://user-images.githubusercontent.com/6771947/192723268-c9b4840d-0f0a-42cd-beac-2dd97e5abbd2.png)

Reply:
It's not a good fit for water or fuel consumption though.

Reply:
Let's hold off an icon and let's see over time what is being used by integrations.

Reply:
Yes, it fits an amount of liquid in a container, but it doesn't fit the amount of water you've used in your home, or the amount of fuel your car has used.